lib/searchable_select.ex

```diff
@@ -47,6 +47,8 @@ defmodule SearchableSelect do
     socket =
       socket
       |> assign(:search, "")
+      |> assign(:disabled, assigns[:disabled] || false)
+      |> assign(:placeholder, assigns[:placeholder] || "Search")
```

Comment:
Already defaulted in the initial update function. A good use case for `assign_new`